mobile/src/common/enums/notification/notification-messages/notification-message.enum.ts

```diff
@@ -0,0 +1,6 @@
+enum NotificationMessage {
+  INTERVIEW_CREATE = 'Interview successfully created!',
+  MENTOR_ADD = 'You are the mentor of this course!',
```

Comment:
Use same messages as on the frontend, also remove notification-messages folder, path should be enum/notification/notification-message.enum.ts